vowpalwabbit/no_label.cc

```diff
@@ -42,7 +42,8 @@ label_parser no_label_parser = {
     [](polylabel& /* label */, reduction_features& /* red_features */, VW::label_parser_reuse_mem& /* reuse_mem */,
         const VW::named_labels* /* ldict */, const std::vector<VW::string_view>& words) { parse_no_label(words); },
     // cache_label
-    [](const polylabel& /* label */, const reduction_features& /* red_features */, io_buf& /* cache */) {},
+    [](const polylabel& /* label */, const reduction_features& /* red_features */, io_buf& /* cache */,
+        const std::string&, bool) -> size_t { return 1; },
```

Comment:
return 1 or 0?

Reply:
I used 1 for consistency since read_cached_label returns 1, they should probably both return 0 though